src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1003,6 +1003,7 @@ public static function redirect($strLocation, $intStatus=303)
 			exit;
 		}
 
+		$strLocation = ltrim($strLocation, '/');
```

Comment:
Sorry, but this must be wrong. If the input contains a leading slash, it's an absolute URL (probably generated by the Symfony router). Removing the leading slash does not (always) correctly convert an absolute to a relative URL! We had exactly the same problems with TL_SCRIPT.